libs/designsystem/dropdown/src/dropdown.component.spec.ts

```diff
@@ -1143,4 +1144,41 @@ describe('DropdownComponent', () => {
       });
     });
   });
+
+  describe('when configured with expand=block and usePopover=true', () => {
+    const createHost = createHostFactory({
+      component: DropdownComponent,
+      imports: [ItemModule],
+      declarations: [
+        ItemComponent,
+        MockComponents(ButtonComponent, CardComponent, IconComponent, IonItem, PopoverComponent),
+      ],
+    });
+
+    let spectator: SpectatorHost<DropdownComponent>;
+
+    beforeEach(() => {
+      spectator = createHost(`<kirby-dropdown></kirby-dropdown>`, {
+        props: {
+          items: items,
+          expand: 'block',
+          usePopover: true,
+        },
+      });
+    });
+
+    it('should update popover card size when resized', async () => {
+      const initWidth = spectator.element.clientWidth;
+      const popoverCard = spectator.element.querySelector<HTMLElement>('kirby-card');
+      const initCardWidth = popoverCard.style.getPropertyValue('--kirby-card-width');
+      expect(initCardWidth).toEqual(`${initWidth}px`);
+
+      const newWidth = '200px';
+      (spectator.hostElement as HTMLElement).style.width = newWidth;
+      await TestHelper.waitForResizeObserver();
```

Comment:
Could the failing test be a timing issue around here? The new width does not seem to have applied yet at this point:
<img width="1041" alt="image" src="https://github.com/kirbydesign/designsystem/assets/42470636/af0ba8cd-5d7b-461f-a3ed-4d351b0b698f">

Reply:
Added a check for the updated width before asserting 🤞🏻